linux-installer/README.md

```diff
@@ -49,44 +49,21 @@ If you have any issues with the installer, try running with the `--trace` argume
 curl -sL https://ibm.biz/idt-installer | bash -s -- --trace
 ```
 
-If updating an existing IDT installation, you can run the following:
+If you wish to update the IBM Cloud Dev Plugin, run:
 ```
-idt update --trace
+ibmcloud plugin update dev --trace
 ```
 
 
 ## Updating
 
-If you wish to update the IBM Developer Tools CLI, run `ibmcloud dev update`. There is also a `--force` argument that will force update to all dependencies too.
+If you wish to update the IBM Cloud Dev Plugin, run `ibmcloud plugin update dev`. There is also a `--force` argument that will force update to all dependencies too.
 
 Syntax:
 ```
-ibmcloud dev update [--force] [--trace]
+ibmcloud plugin update dev [--force] [--trace]
```

Comment:
`--force` and `--trace` aren't valid parameters for the **plugin update** command.